assets/js/theme/account.js

```diff
@@ -201,6 +208,103 @@ export default class Account extends PageManager {
         });
     }
 
+    initPaymentMethodFormValidation($paymentMethodForm) {
+        // Inject validations into form fields before validation runs
+        $paymentMethodForm.find('#first_name.form-field').attr('data-validation', `{ "type": "singleline", "label": "${this.context.firstNameLabel}", "required": true, "maxlength": 0 }`);
```

Comment:
Hey @junedkazi , we have removed the dynamic input fields for the static components. however, I had to inject the validation through javascript because it was breaking the template engine.